### DISCUSSION TITLE
just reminding you, this project may take years to finish

### DISCUSSION THREAD

**Discussion by markverb1** [FlagClicked/FlagClicked#31] [General]
# ⏲just reminding you, this project may take a year to finish  
✅we've only got these things finished
- [x] 🟩node.js setup
- [x] 💻basic api functions
❌here is what we don't have finished
- [x] 👤account creation system
- [ ] 🛠moderation

- [ ] 💬 functions like upvotes, comments
- [ ] 🔎getting found on google
- [ ] 📄basic page layout
- [ ] ❓and much much more
📟so... the project is currently dead. but, we can make it alive again!

**Comment by danielgallab** [comment]
it's not dead, it's just that most people don't know about this project and/or don't know how to help out here. I'm a bit busy but can help soon. beta version will have very little of those things.

**Reply by markverb1** [reply]
yea i guess

**Comment by FunctionalMetatable** [comment]
I'd help you guys but i don't know vue

**Reply by markverb1** [reply]
> I'd help you guys but i don't know vue

Same. But I still contribute. You mostly just need to know JS basics, HTML and CSS.

**Reply by danielgallab** [reply]
Vue is super easy to learn, but you don't need to really learn vue to help.

**Reply by markverb1** [reply]
> Vue is super easy to learn, but you don't need to really learn vue to help.

Exactly.
